tests/test_pages_price.py

```diff
@@ -62,21 +62,21 @@ def price(self):
     url = "https://example.com"
     page = CustomProductPage(response=HttpResponse(url=url, body=html))
     assert page.call_count == 0
-    assert page.price == "$13.2"
+    assert page.price == "13.20"
     assert page.call_count == 1
     assert page.currency is None
-    assert await page.currencyRaw is None
-    assert page.call_count == 2  # we want this to be 1
-    assert await page.currencyRaw is None
-    assert page.call_count == 2  # we want this to be 1
+    assert await page.currencyRaw == "$"
+    assert page.call_count == 1  # we want this to be 1
```

Comment:
I wonder why did the count change and what should be the new expectation in the comment.

Reply:
Yes, I've noticed that comment info doesn't match test values in places in master. Not sure if this is a proper behavior.

On master such difference can be found in 5 places:
* https://github.com/zytedata/zyte-common-items/blob/main/tests/test_pages_price.py#L41 
* https://github.com/zytedata/zyte-common-items/blob/main/tests/test_pages_price.py#L69
* https://github.com/zytedata/zyte-common-items/blob/main/tests/test_pages_price.py#L71
* https://github.com/zytedata/zyte-common-items/blob/main/tests/test_pages_price.py#L80
* https://github.com/zytedata/zyte-common-items/blob/main/tests/test_pages_price.py#L188

This PR fixes two of these:
* https://github.com/zytedata/zyte-common-items/blob/6620a1538ee3b558d78705800ad4ea19f058f7dc/tests/test_pages_price.py#L69
* https://github.com/zytedata/zyte-common-items/blob/6620a1538ee3b558d78705800ad4ea19f058f7dc/tests/test_pages_price.py#L71
While three are same as before:
* https://github.com/zytedata/zyte-common-items/blob/6620a1538ee3b558d78705800ad4ea19f058f7dc/tests/test_pages_price.py#L41
* https://github.com/zytedata/zyte-common-items/blob/6620a1538ee3b558d78705800ad4ea19f058f7dc/tests/test_pages_price.py#L80
* https://github.com/zytedata/zyte-common-items/blob/6620a1538ee3b558d78705800ad4ea19f058f7dc/tests/test_pages_price.py#L188

It seems like an improvement, albeit incomplete (but fixing this for all cases seems kind of out of scope so perhaps we can live with it for now...?)